canonical/lxd-ui
# Autofocus project selector input on open of project list

## Done

- Autofocus project selector input on open of project list

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - open project selector in side navigation with > 5 projects. It should focus on the search input field automatically now, a quality of life improvement to the element
